corehq/messaging/tasks.py

```diff
@@ -58,7 +58,7 @@ def _sync_case_for_messaging(domain, case_id):
 
 def _get_cached_rule(domain, rule_id):
     rules = AutomaticUpdateRule.by_domain_cached(domain, AutomaticUpdateRule.WORKFLOW_SCHEDULING)
-    rules = list(filter(lambda rule: rule.pk == rule_id, rules))
+    rules = list([rule for rule in rules if rule.pk == rule_id])
```

Comment:
nit: you can drop the `list()` on this one

Reply:
good point!